gluecodium/src/main/resources/templates/ffi/FfiBlobHandleHeader.mustache

```diff
@@ -0,0 +1,44 @@
+{{!!
+  !
+  ! Copyright (C) 2016-2019 HERE Europe B.V.
+  !
+  ! Licensed under the Apache License, Version 2.0 (the "License");
+  ! you may not use this file except in compliance with the License.
+  ! You may obtain a copy of the License at
+  !
+  !     http://www.apache.org/licenses/LICENSE-2.0
+  !
+  ! Unless required by applicable law or agreed to in writing, software
+  ! distributed under the License is distributed on an "AS IS" BASIS,
+  ! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ! See the License for the specific language governing permissions and
+  ! limitations under the License.
+  !
+  ! SPDX-License-Identifier: Apache-2.0
+  ! License-Filename: LICENSE
+  !
+  !}}
+{{>ffi/FfiCopyrightHeader}}
+
+#pragma once
+
+#include "Export.h"
+#include "OpaqueHandle.h"
+#include <cstdint>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+_GLUECODIUM_FFI_EXPORT FfiOpaqueHandle blob_create_handle();
+_GLUECODIUM_FFI_EXPORT void blob_release_handle(FfiOpaqueHandle handle);
+_GLUECODIUM_FFI_EXPORT void blob_insert(FfiOpaqueHandle handle, uint8_t value);
```

Comment:
There is no concept of a byte array in Dart where you could pass a raw pointer and length instead?

Reply:
There's a ByteBuffer class: https://api.dart.dev/stable/2.7.1/dart-typed_data/ByteBuffer-class.html, but it does not seem to be intended for non-system use, as there's no sane way to create it.

There's also Uint8List, fixed-size List implementation optimized for 8-bit ints: https://api.dart.dev/stable/2.7.1/dart-typed_data/Uint8List-class.html. I think I should use it here, thanks for the hint.